js/ui/widget/ui.widget.js

```diff
@@ -1,30 +1,25 @@
-var $ = require("../../core/renderer"),
-    eventsEngine = require("../../events/core/events_engine"),
-    events = require("../../events/"),
-    Action = require("../../core/action"),
-    extend = require("../../core/utils/extend").extend,
-    inArray = require("../../core/utils/array").inArray,
-    each = require("../../core/utils/iterator").each,
-    commonUtils = require("../../core/utils/common"),
-    typeUtils = require("../../core/utils/type"),
-    DOMComponentWithTemplate = require("../../core/dom_component_with_template"),
-    selectors = require("./selectors"),
-    eventUtils = require("../../events/utils");
-
-require("../../events/click");
-require("../../events/hover");
-require("../../events/core/emitter.feedback");
-
-const { hover, focus, active, dxClick, keyboard } = events;
-
-var WIDGET_CLASS = "dx-widget",
-    ACTIVE_STATE_CLASS = "dx-state-active",
-    DISABLED_STATE_CLASS = "dx-state-disabled",
-    INVISIBLE_STATE_CLASS = "dx-state-invisible",
-    HOVER_STATE_CLASS = "dx-state-hover",
-    FOCUSED_STATE_CLASS = "dx-state-focused",
-    FEEDBACK_SHOW_TIMEOUT = 30,
-    FEEDBACK_HIDE_TIMEOUT = 400;
```

Comment:
Should we save these types, but define them as constants? Maybe in separate file.

Reply:
Most of these use once ) I think it is not necessary.

Reply:
As I see, we have similar types in other components (for example, `ACTIVE_STATE_CLASS`, `DISABLED_STATE_CLASS`, `INVISIBLE_STATE_CLASS`).  Maybe we should extract them in `constants.js` or something like that.

Reply:
In other PR of course.

Reply:
I don't see the real benefits of this